src/coreclr/src/jit/importer.cpp

```diff
@@ -6484,6 +6475,20 @@ void Compiler::impCheckForPInvokeCall(
 
         // PInvoke CALLI in IL stubs must be inlined
     }
+    else if (IsTargetAbi(CORINFO_CORECLR_ABI) &&
+                (mflags & CORINFO_FLG_PINVOKE) != 0 &&
+                (mflags & CORINFO_FLG_FORCEINLINE) != 0 &&
+                opts.jitFlags->IsSet(JitFlags::JIT_FLAG_IL_STUB) &&
+                opts.ShouldUsePInvokeHelpers())
+    {
```

Comment:
Ideally, I'd like to have a JIT flag for this. Thoughts?

Reply:
What would the meaning of this flag would be?

IMHO, it is what the `JIT_FLAG_IL_STUB` flag is for.

Reply:
Ok, doesn't seem like we need any other flag than `JIT_FLAG_IL_STUB`